week-3/todo-list/index.html

```diff
@@ -3,24 +3,31 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Title here</title>
+    <title>To-do List App</title>
     <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
   </head>
   <body>
+    <h1>TO-DO LIST APP</h1>
     <form>
+      <h3>Write down the things you want to do. ✍️</h3>
```

Comment:
This is the second heading on the page. What would be a better semantic html tag for it as opposed to <h3>?